code/game/objects/items/weapons/grenades/clowngrenade.dm

```diff
@@ -84,18 +86,6 @@
 				M.take_overall_damage(0, max(0, (burned - 2)))
 
 		if(!istype(M, /mob/living/carbon/slime) && !isrobot(M))
-			M.stop_pulling()
```

Comment:
I understand that you deleted this code block because it's inheriting slipping from the parent now, but this slip behavior is actually different (and more intense) than the parent's, so you need to retain it

I think the best way to do this would to be to break off the slipping behavior into a new proc `handle_slip` and just override handle_slip for the traitor peel and bluespace peel. That way you can call the parent crossed without having to double the slippage or remove one unique type of slip.

Reply:
Done.